gui/choc_WebView.h

```diff
@@ -1097,7 +1117,21 @@ MIDL_INTERFACE("15e1c6a3-c72a-4df3-91d7-d097fbec6bfd")
 ICoreWebView2PermissionRequestedEventHandler : public IUnknown
 {
 public:
-     virtual HRESULT STDMETHODCALLTYPE Invoke(ICoreWebView2*, ICoreWebView2PermissionRequestedEventArgs*) = 0;
+    virtual HRESULT STDMETHODCALLTYPE Invoke(ICoreWebView2*, ICoreWebView2PermissionRequestedEventArgs*) = 0;
```

Comment:
non-blocking: this deleted space can be added back

Reply:
ah, weird. this was a tab and a space, but I guess that's upstream 🤷